test-network/prometheus-grafana/README.md

```diff
@@ -0,0 +1,47 @@
+# Description
+
+This sample provides an environment to display and capture metrics from the test-network in real time. It consists of a docker-compose file that starts a prometheus and grafana server setup configured to collect and display metrics for the test network.
+
+# Requirements
+
+This sample is meant to be used on **linux** in order to fully benefit from its capabilities. However, it can be used and adapted on other systems with some additional work and tweaks.
```

Comment:
Probably best to say that it has been tested on Linux only.  If you say 'it can be used' then it just leads to the question - how can it be used?